samples: led (#5)
NordicSemiconductor/npmx-zephyr: samples/led/README.rst

.. _npmx_led_sample:

npmx: LED
#########

.. contents::
   :local:
   :depth: 2

> The npmx LED sample demonstrates how to control PMIC LEDs using npmx drivers.

Reviewer: ```suggestion
The LED sample demonstrates how to control LEDs using npmx drivers.
```

.. _led_sample:

LED
###

.. contents::
   :local:
   :depth: 2

> The LED sample demonstrates how to control LEDs using npmx drivers.

Reviewer 2: ```suggestion
The LED sample demonstrates how to control LEDs with the use of npmx drivers.
```

       - nRF5340 DK pins
       - nRF52840 DK pins
     * - GPIO0
       - P1.10
       - P1.10
     * - SDA
       - P1.02
       - P0.26
     * - SCL
       - P1.03
       - P0.27
     * - VOUT2 & GND
       - External supply (P21)
       - External supply (P21)

#. Make the following connections on the nPM1300 EK:

   * Connect a USB power supply to the **J3** connector.
   * Connect a suitable battery to the **J2** connector.
   * On the **P18** pin header, connect **VOUT2** and **VDDIO** pins with a jumper.
   * On the **P2** pin header, connect **VBAT** and **VBATIN** pins with a jumper.
   * On the **P17** pin header, connect all LEDs with jumpers.
   * On the **P13** pin header, connect **RSET1** and **GND** pins with a jumper.
   * On the **P14** pin header, connect **RSET2** and **VSET2** pins with a jumper.

#. Make the following connections on the chosen DK:

   * Set the **SW9** nRF power source switch to **VDD**.
   * Set the **SW10** VEXT → nRF switch to **ON**.

Configuration
*************

|config|

Building and running
********************

.. |sample path| replace:: :file:`samples/led`

.. include:: /includes/build_and_run.txt

Testing
=======

|test_sample|

> #. |connect_kit|
> #. |connect_terminal|
> #. See blinking LEDs on the board.

Reviewer 2: ```suggestion
#. |connect_kit|
#. |connect_terminal|

As a result, you should see LEDs blinking on the board.
```

> #. See blinking LEDs on the board.

Author: It is just a result.

> #. See blinking LEDs on the board.

Reviewer 2: If so, I would not treat it as a step to be done. Please check my suggestion below.
